cfgov/unprocessed/js/modules/ExternalSite.js

```diff
@@ -24,6 +24,7 @@ function ExternalSite( element ) {
   var _dom = atomicHelpers.checkDom( element, BASE_CLASS );
   var _durationEl = _dom.querySelector( '.external-site_reload-container' );
   var _directEl = _dom.querySelector( '.external-site_proceed-btn' );
+  var _form = _dom.querySelector( 'form#proceed' );
```

Comment:
- Could be `_formEl` to fit with the format of the variables immediately preceding it. 

- Also, you could define the form dom reference inside `_gotoUrl()` instead of globally inside the module, since `_gotoUrl()` is the only place it's referenced and since it's referenced only once per page load.